src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -97,6 +100,8 @@ public void getConceptChildren(CloudPlatform cloudPlatform) {
     CloudPlatformWrapper cloudPlatformWrapper = CloudPlatformWrapper.of(cloudPlatform);
     var concepts = List.of(new SnapshotBuilderConcept().name("concept1").id(1));
     if (cloudPlatformWrapper.isGcp()) {
+      DatasetModel datasetModel = new DatasetModel().name("name").dataProject("project");
+      when(datasetService.retrieveModel(eq(dataset), any())).thenReturn(datasetModel);
```

Comment:
Could you use the `TEST_USER` pattern here, rather than `any()`?

```
private static final AuthenticatedUserRequest TEST_USER =
      AuthenticationFixtures.randomUserRequest();
```